org.eclipse.editorconfig.ui/src/main/java/org/eclipse/editorconfig/ui/internal/EditorConfigVisitor.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017 Nathan Jones, Jackson Bailey
```

Comment:
I've merged my fixes for the functional test suite on Windows. You can rebase this branch on master.